src/lib/ext-monitor.js

```diff
@@ -81,6 +85,8 @@ export default class ExtensionMonitor {
     sendAllLogs: () => ({ existingLogs: this.logs }),
     loadLogs: (requestParams) => this.loadLogs(requestParams),
     getLoadedLogs: (requestParams) => this.getLoadedLogs(requestParams),
+    addPanelTabId: ({ tabId }) => this.devToolsPanelTabIds.add(tabId),
+    deletePanelTabId: ({ tabId }) => this.devToolsPanelTabIds.delete(tabId),
```

Comment:
The `addPanelTabId` and `deletePanelTabId` acts when an ActivityLog Devtools panel is shown or hidden respectively.